timesketch/ui/graphs/graph-view.data.ts

```diff
@@ -59,45 +59,56 @@ export const style = ({nodeLabel, edgeLabel}): CytoscapeStyle => [
       'shape': 'roundrectangle',
       'width': 'label',
       'height': 'label',
-      'padding': 10,
+      'compound-sizing-wrt-labels': 'include',
       'label': (node) => nodeLabel(node.data()),
       'text-halign': 'center',
       'text-valign': 'center',
       'color': '#FFFFFF',
       'font-size': '10',
-      'background-color': '#68BDF6',
-      'border-color': '#5CA8DB',
-      'border-width': '1',
+      'font-weight': 'bold',
+      'text-outline-width': '3px',
+      'padding': '3px',
```

Comment:
I would like to see how 5px would look like here. Just a minor thing though :)

Reply:
![3px](https://user-images.githubusercontent.com/9804230/31942982-77cf59c8-b8c7-11e7-8d50-292bdc4ec1a6.png)
![4px](https://user-images.githubusercontent.com/9804230/31942983-77f625f8-b8c7-11e7-9b82-66dd491efb98.png)
![5px](https://user-images.githubusercontent.com/9804230/31942984-780c15fc-b8c7-11e7-9a08-30b5afede3d5.png)

Reply:
3, 4 and 5 px respectively. I changed it to 4.